test/formatters/stylishFormatterTests.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2013 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+
+import {IFormatter, RuleFailure, TestUtils} from "../lint";
+
+describe("Stylish Formatter", () => {
+    const TEST_FILE = "formatters/stylishFormatter.test.ts";
+    let sourceFile: ts.SourceFile;
+    let formatter: IFormatter;
+
+    before(() => {
+        const Formatter = TestUtils.getFormatter("stylish");
+        sourceFile = TestUtils.getSourceFile(TEST_FILE);
+        formatter = new Formatter();
+    });
+
+    it("formats failures", () => {
+        const maxPosition = sourceFile.getFullWidth();
+
+        const failures = [
+            new RuleFailure(sourceFile, 0, 1, "first failure", "first-name"),
+            new RuleFailure(sourceFile, 2, 3, "&<>'\" should be escaped", "escape"),
+            new RuleFailure(sourceFile, maxPosition - 1, maxPosition, "last failure", "last-name"),
+            new RuleFailure(sourceFile, 0, maxPosition, "full failure", "full-name"),
+        ];
+        const expectedResult = (require("colors").supportsColor) ?
```

Comment:
Can we import colors with an ES6 `import` statement at the top of the file instead of here?